test/index.spec.js

```diff
@@ -300,4 +300,15 @@ describe('CID', () => {
       })
     })
   })
+
+  describe('buffer reuse', () => {
+    it('should cache CID as buffer', done => {
+      multihashing(Buffer.from(`TEST${Date.now()}`), 'sha2-256', (err, hash) => {
```

Comment:
No need to change it, just a general note. I think I'll be moving away from tests that have non-deterministic things in it. It makes things hard to debug if you compare to different run (I recently debugged an issue where I was comparing two different repos and it's way easier if the contents is the same between runs).